lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
         def serializable_hash(options = nil)
           options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
```

Comment:
I think, in general, we should move all customizations to the constructor, and treat `serializable_hash` as if it were `as_json` or `to_json`, which is called by the Renderer, not at the controller when we call `render json: object`.